Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -25,7 +25,8 @@ class BrowserViewController: UIViewController {
     private var webViewContainer: UIView!
     private let uriFixup = URIFixup()
     private var screenshotHelper: ScreenshotHelper!
-    private var homePanelIsInline: Bool = false
+    private var homePanelIsInline = false
+    private var searchLoader: SearchLoader!
```

Comment:
Same reason the other variables here are: we want to use `let`, but we can't since we have an init helper.

We need a shared instance that can be used by both the URL bar and the SVC, so we need one class-level loader to reuse the query.